web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v3110/migrate-oracle.sql

```diff
@@ -16,7 +16,7 @@ INSERT INTO StatusValuesDes  (iddes, langid, label) VALUES (63,'cat','Record res
 INSERT INTO StatusValuesDes  (iddes, langid, label) VALUES (63,'chi','Record restored.');
 INSERT INTO StatusValuesDes  (iddes, langid, label) VALUES (63,'dut','Record restored.');
 INSERT INTO StatusValuesDes  (iddes, langid, label) VALUES (63,'eng','Record restored.');
-INSERT INTO StatusValuesDes  (iddes, langid, label) VALUES (63,'fre','Fiche restaur�.');
+INSERT INTO StatusValuesDes  (iddes, langid, label) VALUES (63,'fre','Fiche restaur�.');
```

Comment:
Did your editor change the encoding?  Not sure why you would be doing this change?  I also see similar changes in the other sql files.

Reply:
No that I'm aware, I use always IntelliJ. I'll check why this encoding change.